src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public static final int MAX_BOUND = 101;
+    private Random random = new Random();
+    private ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Ball getRandomBall() {
+        return new Ball(colorSupplier.getRandomColor(),random.nextInt(MAX_BOUND));
```

Comment:
```suggestion
        return new Ball(colorSupplier.getRandomColor(), random.nextInt(MAX_BOUND));
```
Ctrl Alt L for autoformat